settings/Address.setting.php

```diff
@@ -40,34 +40,41 @@
     ],
     'settings_pages' => ['address' => ['section' => 'standardization', 'weight' => 10]],
   ],
-  'address_standardization_userid' => [
+  'address_standardization_key' => [
     'group_name' => 'Address Preferences',
     'group' => 'address',
-    'name' => 'address_standardization_userid',
-    'type' => 'String',
+    'name' => 'address_standardization_key',
+    'type' => 'Text',
     'html_type' => 'text',
     'default' => NULL,
-    'add' => '4.1',
-    'title' => ts('Provider service user ID'),
+    'add' => '6.10',
```

Comment:
I'm not sure what the right way to remove settings is. This is effectively removing and adding a new one.

In terms of upgrade, what will happen is it will silently stop working, because there is no smooth upgrade path and you need to get new credentials. This might be a good use for a status check.

Also it's now 6.11.